src/test/ui/span/issue-39018.stderr

```diff
@@ -2,11 +2,9 @@ error[E0369]: binary operation `+` cannot be applied to type `&'static str`
   --> $DIR/issue-39018.rs:12:13
    |
 12 |     let x = "Hello " + "World!";
-   |             ^^^^^^^^
+   |             ^^^^^^^^ `to_owned()` can be used to create an owned `String` from a string reference. String concatenation appends the string on the right to the string on the left and may require reallocation. This requires ownership of the string on the left: `"Hello ".to_owned()`
```

Comment:
Yea I was considering also doing a length check. Would it be entirely too confusing to add such an arbitrary limit in the emitter? I mean, someone using span_suggestion might be trying to get their suggestion into a label, but the rule is pretty hidden, even if added to the docs